apps/web/src/services/asset-inventory/collector/shared/collector-forms/PluginMetadataForm.vue

```diff
@@ -0,0 +1,103 @@
+<template>
+    <p-data-loader class="plugin-metadata-form"
+                   :loading="state.loading"
+    >
+        <p-json-schema-form v-if="!state.isLoadFailed && !state.loading"
```

Comment:
suggestion: You can control the default slot of PDataLoader component by `data` prop.